xbmc/storage/cdioSupport.h

```diff
@@ -97,6 +97,13 @@ typedef struct TRACKINFO
 }
 trackinfo;
 
+/*! \brief Helper enum class for the MMC tray state
+*/
+enum class CdioTrayStatus
+{
+  CLOSED, ///< The MMC tray state is reported closed
```

Comment:
It's probably cleaner to put the comment on the previous line instead of inline

Reply:
This is the standard way of documenting enums in doxygen

Reply:
You can use the normal `/*! The MMC tray state is reported closed */` style